stan/math/rev/core/vari.hpp

```diff
@@ -221,7 +232,7 @@ class vari_value<T, require_eigen_dense_base_t<T>> : public vari_base {
    * The adjoint of this variable, which is the partial derivative
    * of this variable with respect to the root variable.
    */
-  eigen_map adj_;
+  mutable eigen_map adj_;
```

Comment:
Where do you need to change adjoint map?

Reply:
Oh before when I wasn't using `arena_matrix` just doing `alpha.adj() += ...` was giving me an eigen error about mutating a `const` type. But putting everything in one and using an `arena_matrix` I don't see that anymore so I'll remove this